dali/dfu/dfuserver.cpp

```diff
@@ -169,7 +169,7 @@ int main(int argc, const char *argv[])
     Owned<IReplicateServer> replserver;
     try {
         Owned<IGroup> serverGroup = createIGroupRetry(daliServer.str(),DALI_SERVER_PORT);
-        initClientProcess(serverGroup, DCR_DfuServer, 0, NULL, NULL, stop?(1000*30):MP_WAIT_FOREVER);
+        initClientProcess(serverGroup, DCR_DfuServer, 0, NULL, NULL, stop?(1000*30):MP_WAIT_FOREVER, true);
```

Comment:
does dfuserver need to listen?

Reply:
I don't think so, I will remove.

Reply:
I initially thought so but I don't think that anymore.